### PR TITLE
add vivaldi in the README

### PR DESCRIPTION
**Pull request by ThiagoG8**

### CODE REVIEW

**Comment by ThiagoG8** [comment]
I am waiting for the approval of the PR and it is an honor to contribute to the Omni theme!

File: README.md
Comment (jpedroschmitz) [inline README.md:11]: Hey, thanks for opening the PR!

Could you place Vivaldi on the end of this browser list? We are ordering the themes alphabetically

**Review by jpedroschmitz — CHANGES_REQUESTED** [review]

**Comment by jpedroschmitz** [comment]
> I am waiting for the approval of the PR and it is an honor to contribute to the Omni theme!

Hey, thanks for opening the PR and for contributing to Omni!

I have reviewed it and requested a small change to keep the pattern on the list

**Review by jpedroschmitz — APPROVED** [review]

**Pull request merged**
